_Week5/Group 3/demonstration.do

```diff
@@ -0,0 +1,17 @@
+// Use case 1
```

Comment:
The assignment says to use a global macro and run your `program.do` file to load the program into memory. I know it's pretty obvious, but you should include it at the start of the demonstration file.